source/locale/pt_BR/LC_MESSAGES/faq.po

```diff
@@ -816,15 +829,16 @@ msgstr ""
 "**Usando o flowbroker para pré-processar dados**: eles podem ajudar a processar "
 "e transformar os dados para que possam ser enviados corretamente para o seu "
 "aplicativo por meio de solicitação HTTP ou armazenados em um dispositivo virtual "
-"(que pode ser usado para gerar notificações conforme descrito anteriormente )."
+"(que pode ser usado para gerar notificações conforme descrito anteriormente "
+"Todos esses endpoints devem ter um token de acesso,)."
 
 #: ../../source/faq/faq.rst:455
 msgid ""
 "All these endpoints should bear an access token, see more :ref:`Getting access "
 "token`."
 msgstr ""
-"Todos esses *endpoints* devem ter um *token* de acesso, veja mais :ref:`Getting "
-"access token`."
+"Todos esses *endpoints* devem ter um *token* de acesso. Para isso, confira :ref:"
+"`Obtendo um token de acesso.`."
```

Comment:
> \`Obtendo um token de acesso`.`\`.

Is it necessary to keep the penultimate full stop?

Reply:
removed penultimate full stop